rules/jest.js

```diff
@@ -7,5 +7,9 @@ module.exports = {
     'jest/no-try-expect': 'off',
     'jest/prefer-called-with': 'off',
     'jest/prefer-inline-snapshots': 'off',
+    'jest/prefer-to-be-null': 'error',
+    'jest/prefer-to-be-undefined': 'error',
+    'jest/prefer-to-contain': 'error',
+    'jest/prefer-to-have-length': 'error',
```

Comment:
Actually, I don't think it's necessary at all: https://github.com/ridedott/eslint-config/blob/b4ee10ed7faad4c033d1f0198c576d444229bd61/index.js#L5